### DISCUSSION TITLE
System authentication (fingerprint) before popping up authenticator page

### DISCUSSION THREAD

**Discussion by nandanprakash** [Authenticator-Extension/Authenticator#1055] [Ideas]
to make sure there is no easy access to authenticator one tIme code, can we add system authentication as optional protection?

**Comment by LuckyLuv77** [comment]
***@***.***

**Comment by Sneezry** [comment]
WebAuth doesn't support encryption, that blocks web apps running locally to use finger printer to protect your data.

To a service-client app, WebAuth proves that you have permission to access the data, and the server returns the data to the client. For such a case, the server owns and manages your data.

However, Authenticator runs locally, your data is on your own machine. Thus, WebAuth can't protect your data.
